src/components/Problems/Problems.tsx

```diff
@@ -54,6 +54,7 @@ export const Problems: FC = () => {
   const semesterId = series?.semester ?? -1
   const canSubmit = series?.can_submit ?? false
   const canResubmit = series?.can_resubmit ?? canSubmit
+  const isAfterDeadline = new Date(series?.deadline ?? '') < new Date()
```

Comment:
to si skusal? ci na dateoch je nakodeny operator aby to porovnalo timestampy alebo nieco? 😄 hladal by som nejake isBefore/isAfter funkcie v luxone

Reply:
no a podobne ako v banneri, budeme asi chciet, aby tu nieco tikalo, checkoval by som tie datumy kludne kazdu sekundu. nejaky `useInterval` by som cakal zase v `use-hooks-ts`

Reply:
luxon dokonca odporuca pouzivat obycajne operatory `>` `<=` https://github.com/moment/luxon/issues/108

Reply:
hmm, nice... ale 😀 oni porovnavaju svoje DateTimes, nie JS Dates 😀 tomu by som teda veril viac 😀 takze bude vhodne DateTime.fromISO() a nejake DateTime.now() abo co a tie porovnat (now som si vymyslel, neviem ako sa robi current v luxone, ale predpokladam)